todo_app/templates/index.html

```diff
@@ -4,15 +4,41 @@
 {% block content %}
   <div class="jumbotron">
     <h1 class="display-4">To-Do App</h1>
-    <p class="lead">Just another to-do app.</p>
+    <p class="lead">Ann's version of to-do app.</p>
   </div>
 
   <div class="row">
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
-        <!-- List the todo items here -->
+        {% for item in items %}
+        <li>{{ item.id }} - {{item.title}} - {{item.status}}</li>
+        {% endfor %}
       </ul>
     </div>
   </div>
+
+  <h2>Add new item</h2>
+  <form action="/addTask" method="post">    
+    <div class="row">
+      <label for="name">New task name: </label>
+    </div>
+    <div class="row">
+      <input type="text" name="TaskName" required>
+      <input type="submit" value="Enter">
+    </div>
+  </form>
+
+  <br/>
+  <h2>Mark item complete</h2>
+  <form action="/endTask" method="post">    
+    <div class="row">
+      <label for="name">Completed task number: </label>
+    </div>
+    <div class="row">
+      <input type="text" name="TaskNo" required>
+      <input type="submit" value="Enter">
+    </div>
+  </form>
```

Comment:
Rather than requiring the user to select the task number in this form at the bottom, wouldn't it be nice to just have a button next to each item in the list?

This can be achieved by:
- Moving the form into the for loop
- Changing the TaskNo input to have:
  - `type="hidden"` instead of "text"
  - `value="{{item.id}}"` so that it just takes the id automatically from the item, no user interaction required.
- Delete the label
- Form itself and submit button don't need to be changed but you probably want different text on the button.

So each li has its own form and for each form all the user sees is the submit button.

Reply:
good idea, will try that now.